### PR TITLE
Set timesteps after events

### PR DESCRIPTION
**Pull request by totto82**
1) The time step after an event can either be set using
timestep_in_days_after_event 
or using the TUNING keyword in the deck.

2) A timestep control pid+newtoniteration that adjust the timestep based on number of Newton iterations is made available. 

Default is to not use any of this. So this should not effect any results.

### CODE REVIEW

File: opm/simulators/timestepping/AdaptiveTimeStepping_impl.hpp [hunk 1]
```diff
@@ -89,6 +89,7 @@ namespace Opm {
         , timestep_verbose_( param.getDefault("timestep.verbose", bool(true) ) && terminal_output )
         , suggested_next_timestep_( tuning.getTSINIT(time_step) )
         , full_timestep_initially_( param.getDefault("full_timestep_initially", bool(false) ) )
+        , timestep_after_event_( tuning.getTMAXWC(time_step))
     {
```
Comment (atgeirr) [inline opm/simulators/timestepping/AdaptiveTimeStepping_impl.hpp]: The member use_newton_iteration_ is not initialized here.

File: opm/simulators/timestepping/TimeStepControl.cpp [hunk 2]
```diff
@@ -177,16 +177,20 @@ namespace Opm
...
-        if( iterations > target_iterations_ )
-        {
-            // if iterations was the same or dts were the same, do some magic
-            dtEstimate *= double( target_iterations_ ) / double(iterations);
+        double dtEstimatePID = BaseType :: computeTimeStepSize( dt, iterations, relChange, simulationTimeElapsed);
+
+        // adjust timesteps based on target iteration
+        double dtEstimateIter;
+        if (iterations > target_iterations_) {
+            double percent = double(iterations - target_iterations_) / target_iterations_;
```
Comment (atgeirr) [inline opm/simulators/timestepping/TimeStepControl.cpp]: The variable `percent` is not a percentage, which is a little misleading, `off_target_fraction` or something would perhaps be better?

**Comment by atgeirr** [comment]
Overall this looks fine, with only a few minor issues (the most important is the missing initialization).

Will merge when fixed and tested.

**Comment by totto82** [comment]
Thanks for the review.

**Comment by atgeirr** [comment]
jenkins run norne_ebos please

**Comment by atgeirr** [comment]
Verified, merging.

**Pull request merged**